source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -286,6 +335,15 @@ namespace PlayFabUnit
     {
         eventTestContext = nullptr;
         eventApi = nullptr;
+
+        if (testThreadPool.size() > 0)
+        {
```

Comment:
this size check is unnecessary. if the size is 0 then the loop won't execute and the clear will be a no-op